main.go

```diff
@@ -114,7 +116,10 @@ func main() {
 	go catchListenAndServe(pubAddr, rout, "proxy", wg)
 	logInfo("router: listening for requests on " + pubAddr)
 
-	api := newAPIHandler(rout)
+	api, err := newAPIHandler(rout)
```

Comment:
I think it would be cleaner to pass in `reloadInterval` here.  This will also make it consistent with the `NewRouter` call above.